docs/ref/models/database-functions.txt

```diff
@@ -293,3 +293,518 @@ Usage example::
 .. versionchanged:: 1.9
 
     The ability to register the function as a transform was added.
+
+Date Functions
+==============
+
+.. module:: django.db.models.functions.date
+
+.. versionadded:: 1.10
+
+We'll be using the following model in examples of each function::
+
+    class Experiment(models.Model):
+        start_time = models.DateTimeField()
+        end_time = models.DateTimeField(null=True, blank=True)
+
+``Extract``
+-----------
+
+.. class:: Extract(date_expression, lookup_name=None, **extra)
```

Comment:
The argument is named `expression` in the code. Did you intentionally rename it in the docs or is it an oversight?